src/azul/terra.py

```diff
@@ -356,6 +356,7 @@ def _request(self,
                                              url,
                                              fields=fields,
                                              headers=headers,
+                                             timeout=config.terra_client_timeout,
```

Comment:
Heads up it looks like the urllib3 docs specify timeout to be a float, does not look like its causing issues. 

https://urllib3.readthedocs.io/en/latest/user-guide.html#using-timeouts